railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -14,6 +14,11 @@ Rails.application.configure do
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
+  # Attempt to read encrypted secrets from `config/secrets.yml.enc`.
+  # Requires an encryption key in `ENV["RAILS_MASTER_KEY"]` or
+  # `config/secrets.yml.key`.
+  config.read_encrypted_secrets = true
```

Comment:
Hello @kaspth 😄 

The USAGE file above says:

> A `shared:` top level key is also supported such that any keys there is merged into the other environments.

This leads me to believe that any "shared" secret key will automatically be available in development.

However this is not true: encrypted secrets are currently only read in production.

**How do you feel about changing this and reading encrypted secrets in every environment?**

If you think about it, `config/secrets.yml.enc` is already organized by environment (just like config/secrets.yml), so even if you _read_ it in development, you will not be importing the keys under `production:`, only those under `shared:` and `development:`.